politeiawww/config.go

```diff
@@ -845,3 +858,11 @@ func loadConfig() (*config, []string, error) {
 
 	return &cfg, remainingArgs, nil
 }
+
+func (p *politeiawww) dcrdataHttpApi() string {
```

Comment:
Initialisms should be the same case.

https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Maybe something like dcrdataHostHTTP() and dcrdataHostWS().